skiko/src/awtMain/kotlin/org/jetbrains/skiko/swing/SkiaSwingLayer.kt

```diff
@@ -28,15 +28,14 @@ open class SkiaSwingLayer(
     renderDelegate: SkikoRenderDelegate,
     analytics: SkiaLayerAnalytics = SkiaLayerAnalytics.Empty,
     externalAccessibleFactory: ((Component) -> Accessible)? = null,
+    private val properties: SkiaLayerProperties = SkiaLayerProperties()
```

Comment:
It seems unrelated + breaking compatibility for no reason. Can we avoid it in this PR?

Reply:
This is needed in the test - to pass the graphic API:
https://github.com/JetBrains/skiko/pull/1016/files#diff-f09f8e25b6e613b4acc42736872562fec343ba85cd033a84464e2ee426a1cbd7R188

As for compatibility, we don't support it in Skiko